engine/src/main/java/io/zeebe/engine/state/LastProcessedState.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.state;
+
+import io.zeebe.db.ColumnFamily;
+import io.zeebe.db.DbContext;
+import io.zeebe.db.ZeebeDb;
+import io.zeebe.db.impl.DbLong;
+import io.zeebe.db.impl.DbString;
+
+public final class LastProcessedState {
+
+  private static final String LAST_PROCESSED_EVENT_KEY = "LAST_PROCESSED_EVENT_KEY";
+  private static final long NO_EVENTS_PROCESSED = -1L;
+
+  private final DbString lastProcessedEventKey;
+  private final DbLong lastProcessedEventPosition;
+  private final ColumnFamily<DbString, DbLong> lastProcessedRecordPositionColumnFamily;
+
+  public LastProcessedState(final ZeebeDb<ZbColumnFamilies> zeebeDb, final DbContext dbContext) {
+    lastProcessedEventKey = new DbString();
+    lastProcessedEventKey.wrapString(LAST_PROCESSED_EVENT_KEY);
+    lastProcessedEventPosition = new DbLong();
+    lastProcessedRecordPositionColumnFamily =
+        zeebeDb.createColumnFamily(
+            ZbColumnFamilies.DEFAULT, dbContext, lastProcessedEventKey, lastProcessedEventPosition);
+  }
+
+  public void markAsProcessed(final long position) {
+    lastProcessedEventPosition.wrapLong(position);
+    lastProcessedRecordPositionColumnFamily.put(lastProcessedEventKey, lastProcessedEventPosition);
+  }
+
+  public long getLastSuccessfulProcessedRecordPosition() {
```

Comment:
These method names mirror the methods in ZeebeState but I think here they can be shortened to be less redudant. So instead of `LastProcessedState#getLastSuccessfulProcessedRecordPosition()` something like `LastProcessedState#getPosition`. I think you get the same information from both. What do you think?

Reply:
I think you are right. I will change it.